crates/editor/src/editor.rs

```diff
@@ -2841,7 +2844,22 @@ impl Editor {
 
             drop(snapshot);
             let had_active_inline_completion = this.has_active_inline_completion(cx);
-            this.change_selections(Some(Autoscroll::fit()), cx, |s| s.select(new_selections));
+
+            let trigger_in_words = !had_active_inline_completion;
+            let completions_triggered =
```

Comment:
> let completions_triggered

Do we really need that boolean?
I think if the completions did not trigger here, we do not need to trigger anything below, in `this.selections_did_change` (because what really changes?), and if it did trigger, we're sure should not trigger anything.

So, again would propose to get that back into the `change_selections_inner` method instead, as proposed in https://github.com/zed-industries/zed/pull/11875#pullrequestreview-2061392229